lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -116,6 +129,8 @@ class AwsCompileEventBridgeEvents {
             const Input = event.eventBridge.input;
             const InputPath = event.eventBridge.inputPath;
             let InputTransformer = event.eventBridge.inputTransformer;
+            let RetryPolicy = event.eventBridge.retryPolicy;
```

Comment:
Is there a reason to assign `event.eventBridge.retryPolicy` to this variable? Could we just leave it undefined and assign an object to it if `event.eventBridge.retryPolicy` is defined? 

Same for `DeatLetterConfig`